Makefile

```diff
@@ -183,7 +158,6 @@ docker-push: ## Push docker image with the manager.
 
 .PHONY: install
 install: manifests kustomize ## Install CRDs into the K8s cluster specified in ~/.kube/config.
-	$(KUSTOMIZE) build config/crd | kubectl apply -f -
```

Comment:
We were getting error because there is nothing inside bases folder(kubectl apply throws an error). 
I think we should remove it for now with a TODO comment to add this statement when we have a CRD inside bases folder.
`$ kustomize build config/crd | kubectl apply -f -
error: no objects passed to apply
`